src/test/java/org/opensearch/neuralsearch/common/BaseNeuralSearchIT.java

```diff
@@ -28,14 +34,20 @@
 import org.opensearch.client.Response;
 import org.opensearch.client.RestClient;
 import org.opensearch.client.WarningsHandler;
+import org.opensearch.common.Strings;
+import org.opensearch.common.xcontent.ToXContent;
+import org.opensearch.common.xcontent.XContentBuilder;
 import org.opensearch.common.xcontent.XContentFactory;
 import org.opensearch.common.xcontent.XContentHelper;
 import org.opensearch.common.xcontent.XContentType;
-import org.opensearch.test.rest.OpenSearchRestTestCase;
+import org.opensearch.index.query.QueryBuilder;
+import org.opensearch.knn.index.SpaceType;
+import org.opensearch.neuralsearch.OpenSearchSecureRestTestCase;
+import org.opensearch.rest.RestStatus;
 
 import com.google.common.collect.ImmutableList;
 
-public abstract class BaseNeuralSearchIT extends OpenSearchRestTestCase {
+public abstract class BaseNeuralSearchIT extends OpenSearchSecureRestTestCase {
 
     private static final Locale LOCALE = Locale.getDefault();
```

Comment:
Not a issue for this PR. Looks like Locale.getDefault() seem to have issue with Windows shall we use Locale.ROOT? Take it in other PR

Reply:
Let use Locale.ROOT.

I think you have not rebased the code, this statement has been updated.